INSTRUCTIONS.native.md

```diff
@@ -0,0 +1,92 @@
+The following instructions are for setting up MSVC without docker.
+
+# FAQ
+
+## Does it run on Ubuntu 18.04 LTS?
+
+Yes, but the install scripts won't work because `msitools` is too old. You'll need to install either via Docker or on a real Ubuntu 20.04 LTS machine; and later copy paste the files under `/opt/msvc`
+
+## Does it work with CMake?
+
+Yes, but a custom version is needed or else CMake will complain that `/opt/msvc/bin/x64/cl` can't build a simple program.
```

Comment:
IIRC there's an issue with discussion about this as well, with indications that it might work kinda fine with stock CMake too - it might be good to link to that too.

Reply:
I noticed the comment about vanilla CMake working but I tried it on Cmake version 3.22.1 and it was still broken.

I didn't try but I suspect that if I do the first CMake with the modified version (so that CMake compiles the test app) afterwards any later configure will work with any cmake version (this is a theory, I didn't try it yet)